### DISCUSSION TITLE
遗忘了账号密码登录不了

### DISCUSSION THREAD

**Discussion by element-xuan** [lsky-org/lsky-pro#714] [问答]
有没有一种办法直接更改数据库中的密码先让我登录上去在自行更改密码，现在用户登录不上，找回密码显示错误，没有办法登录到设置界面

**Comment by 0xxb** [comment]
https://github.com/lsky-org/lsky-pro/issues/498#issuecomment-1203568989

**Reply by element-xuan** [reply]
非常感谢您的帮助，我可以出一份类似的教程吗？因为我在网上搜索了很多但是没有找到过，因此才来此提问浪费了您的时间
